controlpanel/api/aws.py

```diff
@@ -426,8 +423,8 @@ def grant_group_bucket_access(group_policy_arn, bucket_arn, access_level, path_a
     policy.put()
 
 
-def revoke_group_bucket_access(group_policy_arn, bucket_arn=None, path_arns=[]):
-    if not bucket_arn and not path_arns:
+def revoke_group_bucket_access(group_policy_arn, bucket_arn=None):
```

Comment:
Again, I didn't dive deep into this but from quick look it may be possible to make `bucket_arn` mandatory and remove the `if not bucket_arn` and associated warning.

Is it even possible to have a `RoleGroup` instance without an ARN? 😕